docs/pages/database-access/guides/clickhouse-self-hosted.mdx

```diff
@@ -0,0 +1,250 @@
+
+Additionally, your ClickHouse database user accounts must be configured to require a valid client certificate.
+
+```sql
+CREATE USER alice IDENTIFIED WITH ssl_certificate CN 'alice';
+```
+
+By default, the created user may not have access to anything and won't be able to connect, so let's grant it some permissions:
+
+```sql
+GRANT ALL ON *.* TO alice;
+```
+
+## Step 4/5. Configure and Start the Database Service
+Install and configure Teleport where you will run the Teleport Database Service:
+
+<Tabs>
+<TabItem label="Linux Server">
+
+(!docs/pages/includes/install-linux.mdx!)
+
+<Tabs>
+  <TabItem label="ClickHouse HTTP protocol">
+
+Configure and start the Teleport Service:
+
+(!docs/pages/includes/database-access/db-configure-start.mdx dbProtocol="clickhouse" dbName="clickhouse-http" databaseAddress="clickhouse.example.com:8443" !)
+  </TabItem>
+  <TabItem label="ClickHouse Native protocol">
+
+Configure and start the Teleport Service:
+
+(!docs/pages/includes/database-access/db-configure-start.mdx dbProtocol="clickhouse" dbName="clickhouse" databaseAddress="clickhouse.example.com:9440" !)
+  </TabItem>
+</Tabs>
+
+</TabItem>
+<TabItem label="Kubernetes Cluster">
+  Teleport provides Helm charts for installing the Teleport Database Service in Kubernetes Clusters.
+
+  (!docs/pages/kubernetes-access/helm/includes/helm-repo-add.mdx!)
+
+<Tabs>
+  <TabItem label="ClickHouse HTTP Protocol">
+
+(!docs/pages/includes/database-access/db-helm-install.mdx dbName="clickhouse-http" dbProtocol="clickhouse-http" databaseAddress="clickhouse.example.com:8443" dbName="clickhouse-http" !)
+
+  </TabItem>
+  <TabItem label="clickHouse Native Protocol">
+
+(!docs/pages/includes/database-access/db-helm-install.mdx dbName="clickhouse" dbProtocol="clickhouse" databaseAddress="clickhouse.example.com:9440" dbName="clickhouse" !)
+
+  </TabItem>
+</Tabs>
+
+</TabItem>
+</Tabs>
+
+(!docs/pages/includes/database-access/multiple-instances-tip.mdx !)
+
+## Step 5/5. Connect
+
+Once the Database Service has joined the cluster, log in to see the available
+databases:
+
+
+<Tabs>
+  <TabItem label="ClickHouse HTTP Protocol">
+
+```code
+$ tsh login --proxy=teleport.example.com --user=testuser
+$ tsh db ls
+# Name                            Description Allowed Users Labels  Connect
+# ------------------------------- ----------- ------------- ------- ------------------------------
+# clickhouse-http (user: alice)             [*]           env=dev tsh db connect clickhouse-http
+```
+
+<ScopedBlock scope={["cloud"]}>
+
+```code
+$ tsh login --proxy=mytenant.teleport.sh --user=testuser
+$ tsh db ls
+# Name                            Description Allowed Users Labels  Connect
+# ------------------------------- ----------- ------------- ------- ------------------------------
+# clickhouse-http (user: alice)             [*]           env=dev tsh db connect clickhouse-http
+```
+
+</ScopedBlock>
+
+Start Teleport Local Proxy:
```

Comment:
This is the first usage of the term "Local Proxy" in the docs, which shouldn't really a proper noun (it's not a product name, for example). I think we can make this a bit more descriptive about what the `tsh proxy db` command is doing.

Reply:
Updated.